### DISCUSSION TITLE
Status History panel data display

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64458] [Dashboards, Panels & Visualizations]
I am hoping for some improvements to the Status History panel. Running on v9.4.1.

Here I am showing the same data three different ways, with a Time Series panel, State Timeline panel, and Status History panel:
![image](https://user-images.githubusercontent.com/7028403/223804623-bd763899-c5f8-4096-80b8-c1cc226bed24.png)

I like that when it can fit the columns, the Status History shows the discrete data points. I don't like how the State Timeline merges them together. But when there is no data, like here:
![image](https://user-images.githubusercontent.com/7028403/223805178-729a30f1-bd61-4092-bf18-bb29fe954fc4.png)
I think that the Status History should show no data, like the Time Series does, not the last data point.

When there is too much data to show discrete data points, I think the Status History should look like the State Timeline, instead of just refusing to plot anything:
![image](https://user-images.githubusercontent.com/7028403/223805518-0c01b808-455e-4c31-a4b8-0d8ac70bb962.png)

My other gripe with the Status History is that when using a time range ending "now", the time axis starts with the time of the first data point in the time range, not at the beginning of the time range like the other dashboards do:
![image](https://user-images.githubusercontent.com/7028403/223806855-151a0577-2b96-4fb5-9751-b9d9a65bbaa6.png)
This can be misleading at a quick glance.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82790. Feel free to continue the discussion around this there. Thank you!
